CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 A race condition in the index logic was able to lead to incomplete or empty
+  result sets for `vast export`. This has been fixed.
```

Comment:
I think something like `This has been fixed` doesn't provide much insightful information. The fact that we are resolving bug should be obvious from the bug icon. (I found one other occurrence of this wording in the changelog, alas.)

I suggest either removing the sentence or giving an idea *how* the bug has been fixed.

Reply:
I opted for removal. The change log is not the place for detailed information.